riva/client/tts.py

```diff
@@ -111,13 +137,15 @@ def synthesize_online(
             audio_prompt_encoding: (:obj:`AudioEncoding`): Encoding of audio prompt file, e.g. ``AudioEncoding.LINEAR_PCM``.
             quality: (:obj:`int`): This defines the number of times decoder is run. Higher number improves quality of generated
                                    audio but also takes longer to generate the audio. Ranges between 1-40.
+            user_dictionary (:obj:`str`, `optional`): A file path to a user dictionary with key-value pairs separated by double spaces.
 
         Yields:
             :obj:`riva.client.proto.riva_tts_pb2.SynthesizeSpeechResponse`: a response with output. You may find
             :class:`riva.client.proto.riva_tts_pb2.SynthesizeSpeechResponse` fields description `here
             <https://docs.nvidia.com/deeplearning/riva/user-guide/docs/reference/protos/protos.html#riva-proto-riva-tts-proto>`_.
             If :param:`future` is :obj:`True`, then a future object is returned. You may retrieve a response from a
             future object by calling ``result()`` method.
+            user_dictionary (:obj:`str`, `optional`): A file path to a user dictionary with key-value pairs separated by double spaces.
```

Comment:
this param shouldn't be in the yield section, right?

Reply:
the param description comment is missing for synthesize() function, can be added at line 68